src/dawn/engine_dawn.h

```diff
@@ -89,6 +89,9 @@ class EngineDawn : public Engine {
   // Mapping from the generic engine's Pipeline object to our own Dawn-specific
   // pipelines.
   std::unordered_map<amber::Pipeline*, ::amber::dawn::Pipeline> pipeline_map_;
+
+  std::unordered_map<uint32_t, ::dawn::Buffer*> buffer_map_;
```

Comment:
In Dawn, is there no concept of descriptor set number?
I only saw "binding" number in the above lines of code.

Reply:
Yes you're right! It's fixed in the new commits.